README.md

```diff
@@ -65,9 +65,9 @@ package hello;
 service HelloService {
   rpc SayHello(HelloRequest) returns (HelloResponse) {
     option (google.api.http) = {
-      get: "/v1/messages/{name}"
+      get: "/v1/messages/{greeting}"
       additional_bindings {
-        get: "/v1/messages/legacy/{name=**}"
+        get: "/v1/messages/legacy/{greeting=**}"
       }
```

Comment:
protobuf encoding discards the field names, so either way should work, as long as it's consistent.  i faintly remember seeing two different `hello.proto` files (probably examples from grpcbin and grpc-web), and both worked but were slightly different, so i chose one for the doc.  maybe the other one is more visible?

in any case, changing this requires also changing the request examples, like @fffonion pointed

Reply:
@javierguerragiraldez I think that was me copy pasting from an example somewhere. But other places like the grpc-web plugin and the spec in kong itself uses `greeting` . So I agree keep them in sync is a good idea.